src/main/java/com/sleekbyte/tailor/format/XcodeFormatter.java

```diff
@@ -74,6 +71,27 @@ public void displayParseErrorMessage() throws IOException {
         System.out.println(inputFile + Messages.COULD_NOT_BE_PARSED);
     }
 
+    @Override
+    public void displaySummary(long numFiles, long numSkipped, long numErrors, long numWarnings) {
+        long numFilesAnalyzed = numFiles - numSkipped;
+        long numViolations = numErrors + numWarnings;
+        System.out.println(String.format("%nAnalyzed %s, skipped %s, and detected %s (%s, %s).%n",
+            Formatter.pluralize(numFilesAnalyzed, "file", "files"),
+            Formatter.pluralize(numSkipped, "file", "files"),
+            Formatter.pluralize(numViolations, "violation", "violations"),
+            Formatter.pluralize(numErrors, "error", "errors"),
+            Formatter.pluralize(numWarnings, "warning", "warnings")
+        ));
+    }
+
+    @Override
```

Comment:
Shouldn't getExitStatus() be implemented in the base class? It is repeated in both XcodeFormatter and JsonFormatter.